resource-managers/kubernetes/README.md

```diff
@@ -17,13 +17,13 @@ important matters to keep in mind when developing this feature.
 To build Spark with Kubernetes support, use the `kubernetes` profile when invoking Maven. For example, to simply compile
 the Kubernetes core implementation module along with its dependencies:
 
-    build/mvn compile -Pkubernetes -pl resource-managers/kubernetes/core -am
+    build/mvn compile -Pkubernetes -pl resource-managers/kubernetes/core -am -DskipTests
 
 To build a distribution of Spark with Kubernetes support, use the `dev/make-distribution.sh` script, and add the
 `kubernetes` profile as part of the build arguments. Any other build arguments can be specified as one would expect when
 building Spark normally. For example, to build Spark against Hadoop 2.7 and Kubernetes:
 
-    dev/make-distribution.sh --tgz -Phadoop-2.7 -Pkubernetes
+    dev/make-distribution.sh --tgz -Phadoop-2.7 -Pkubernetes -DskipTests
```

Comment:
I think, but am not positive, that make-distribution skips tests by default. Let's check the script to see.